TERIMLERSOZLUGU.md

```diff
@@ -51,64 +53,64 @@ Terimler sözlüğü ayrıca Türkçe'ye çevirilmeyecek sözcüklerin de belirl
  * Constructor: 
 
 
-#D
+## D
  * Destructor: İmha eden
  * Docker: 
  * Debugging: Hata Ayıklama
 
 
-#E
+## E
 
 
-#F
+## F
  * Framework: Yapı
 
-#G
+## G
  * Generator: Üreteç
  * Governance: Yönetim Biçimi
  * Governance Model: Yönetim Modeli
 
-#H
+# H
 
 
-#İ
+## İ
  * İterator: Yineleyici

```

Comment:
This should be `iterator` / `Iterator` I guess, considering English has no `İ` letter. That also fixes this ugly `[i](#İ)` situation on line 15.

Reply:
Sorry latency, I updated request.